### ISSUE TITLE
German translation

### ISSUE THREAD

**Issue by DevYorma** [BentoBoxWorld/addon-bskyblock#10]
Hey,
the German translation is incomplete so I receive messages like "commands.island.go.teleport".
It would be very nice if you can fix that.

**Comment by DevYorma** [comment]
At the moment I translate it into German. I can send you my translation if you want.

**Comment by tastybento** [comment]
Yes, please. It should default to English so I think this is a bug that needs fixing too.

**Comment by tastybento** [comment]
Ah, I see the current German translation is completely out of date and uses the wrong YAML. Please use the US English as the basis for the translation. Thank you!

**Comment by DevYorma** [comment]
What should be displayed instead of this variable?
http://prntscr.com/l2xjls

**Comment by DevYorma** [comment]
The US English translation is incomplete too.
You need to add a description for all schem commands.

**Comment by DevYorma** [comment]
When I execute "/is sethome 2" then it says "&cYou don't have the permission to execute this command (&7[permission]&c)." Without changing the placeholder "[permission]".

**Comment by Poslovitch** [comment]
@DevYorma Any news on this one?

**Comment by DevYorma** [comment]
I am nearly finish (only flags need to be translated). I was very bussy last time but I hope that I could finish it next weak.

**Comment by Poslovitch** [comment]
Great, thanks for the work you're putting in it.
However, which german locale file are you doing? BentoBox's? BSkyBlock's?

(if it's BentoBox's I'll move this issue to the appropriate repo)

**Comment by DevYorma** [comment]
I thought that the BentoBox and BSkyBlock local files are one file?
At the moment I translate BentoBox/BSkyBlock's and Level's locale file

By the way:
![grafik](https://user-images.githubusercontent.com/41928283/48316312-62566980-e5e2-11e8-841d-09515e2d6bed.png)

**Comment by Poslovitch** [comment]
BentoBox =/= BSkyBlock. The main thing is that BSkyBlock and AcidIsland are pretty much using 95% of the features provided by BentoBox, and therefore all the localization comes from there.

The %bsb_plugin% thing should have been removed a long time ago. Haven't it been?
